ui/src/components/helpers/userInformationHelpers.js

```diff
@@ -86,7 +86,10 @@ const updateUserDataRequest = async (data) => {
 };
 
 const getPublicFormsRequest = async () => {
```

Comment:
is this properly named? my thought of public is actually is isLoggedIn = false cus public means anyone can access right